tools/gulp/gulpfile.ts

```diff
@@ -0,0 +1,4 @@
+import "./tasks/clean";
```

Comment:
I don't like the fact that we are using deprecated api, and that we are using that kind of imports - this file exports all of tasks, yet we do not explicitly see anything here, it's useless

please use newer api and make those exports explicit in this file